datafusion/physical-plan/src/aggregates/group_values/column_wise.rs

```diff
@@ -36,7 +36,7 @@ use datafusion_physical_expr::binary_map::OutputType;
 
 use hashbrown::raw::RawTable;
 
-/// Compare GroupValue Rows column by column
+/// A [`GroupValues`] that stores multiple columns of group values.
```

Comment:
I'm not very familiar with it and thus the question, what is group values? 
is it group keys, or exact values attached for specific key?

Reply:
Consider the following example, (1, 'a') is one of the group values, so is (2, 'b') and (3, 'c')
```
statement ok
create table t(a int, b varchar) as values (1, 'a'), (2, 'b'), (1, 'a'), (3, 'c');

query ITI
select a, b, count(*) from t group by a, b;
----
1 a 2
2 b 1
3 c 1
```

Reply:
In Rows implementation, we convert (1, 'a') to row and compare against it. In Column implementation, we compare iteratively from 1 and 'a' in this case.

Reply:
This is a great example. I added in to the docs